SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -634,6 +639,43 @@ void USpatialSender::ResolveOutgoingRPCs(UObject* Object)
 	}
 }
 
+FString USpatialSender::GetOwnerWorkerAttribute(AActor* Actor)
+{
```

Comment:
Add a comment on the purpose of this function and reasoning for various codepaths it can take.

Reply:
Added comments.